Model/Order/PlaceOrder.php

```diff
@@ -137,11 +181,234 @@ public function place(string $shopId, OrderDataInterface $order): ResultInterfac
         );
     }
 
+    /**
+     * @inheritDoc
+     */
+    public function authorizeAndPlace(string $publicOrderId, string $quoteMaskId): ResultInterface
+    {
+        $websiteId = (int)$this->storeManager->getStore()->getWebsiteId();
+        $shopId = $this->config->getShopId($websiteId) ?? '';
+
+        try {
+            $quoteId = $this->maskedQuoteIdToQuoteId->execute($quoteMaskId);
+        } catch (NoSuchEntityException) {
+            return $this->getValidationErrorResponse((string)__('Invalid quote mask ID "%1"', $quoteMaskId));
```

Comment:
[Clarification] I'm not familiar with internationalization in magento, are there commands we need to run to confirm these strings have been added or are these all pre-existing responses?

Reply:
Technically, you do not need to do anything to translate anything in Magento, other than wrapping a string with the `__()` function. (Magento will look for a translation if one is available and if not it'll just spit out the raw string.) Best practice, however, is the generate a base translation dictionary for US English and include that with the rest of your extension code. I _did_ point out to Dave, however, that we haven't done this for any of our extensions yet.